docs/supported_apis.md

```diff
@@ -15,4 +15,4 @@
 | DNN API         | [HIP DNN API](tables/CUDNN_API_supported_by_HIP.md)                  |       |        |
 | CUB API         | [HIP CUB API](tables/CUB_API_supported_by_HIP.md)                    |       |        |

```

Comment:
The order is not consistent at the bottom of the list between the topic and the TOC. 
In addition, the items in the first column (CUDA) use names that are not the same as the TOC. For example:
CUBLAS
CURAND
CUDNN

![image](https://github.com/ROCm/HIPIFY/assets/164940351/51a92fde-9377-40ad-b8a2-2c8c38bff533)

Reply:
Fixed.